src/components/Questions/Question.vue

```diff
@@ -99,7 +99,8 @@
 						</template>
 						{{ t('forms', 'Technical name') }}
 					</NcActionInput>
-					<NcActionButton @click="onClone">
+					<NcActionButton :close-after-click="true"
```

Comment:
For boolean properties this is not needed and depends on your preference / styleguide as this is the same like:
```suggestion
					<NcActionButton close-after-click
```